scripts/zipLookupSMS.js

```diff
@@ -0,0 +1,74 @@
+function Event(opts) {
+  for (var key in opts) {
+    this[key] = opts[key]
+  }
+}
+
+var Distance = require('geo-distance')
```

Comment:
Yeah, the intake app isn't using semis in general... was thinking of being consistent so we can use the same linter files... but I am not super consistent since some of the code I copied from other projects has them.